documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -1,33 +1,32 @@
 [[raspi-config]]
 == `raspi-config`
 
-TIP: Raspberry Pi Desktop users can access a graphical version of this application at **Preferences** > **Raspberry Pi Configuration**. However, some advanced configuration is only available in `raspi-config`.
+`raspi-config` helps you configure your Raspberry Pi. Changes to `raspi-config` will affect xref:config_txt.adoc#what-is-config-txt[`/boot/firmware/config.txt`] and various Linux configuration files.
 
-`raspi-config` helps you configure your Raspberry Pi.  Available options may differ between Raspberry Pi models. To open the configuration tool, run the following command:
+=== Getting started
+
+To open the configuration tool from the desktop GUI, go to **Preferences** > **Raspberry Pi Configuration**.
+
+Alternatively, to access the configuration tool via the terminal:
```

Comment:
I'm not sure this is "correct" as it implies that the GUI version and the terminal version are equivalent, when they're not. The GUI version is much easier to use, but as the now-deleted TIP used to say, there are several advanced options which are available in the CLI tool which aren't available in the GUI tool.
IMHO :slightly_smiling_face:

Reply:
After the code snippet, I would add something like this reduced-size tip admonition:

```
TIP: Some advanced configuration, including <common example>, is only available in `raspi-config`.
```

Use your best judgment to determine if there's a decent example worth tossing in there. Personally I think the statement feels a little empty and meaningless without an example -- _what_ kind of 'advanced' config?!?! -- so I lean towards including it if we can come up with one that's short and sweet.